release_notes.md

```diff
@@ -3,18 +3,18 @@
 ## Enhancement
 
 - Introduced New playbook:
-    - **Ingest Latest CISA ICS Advisory**, playbook ingests all the ICS Advisory record.
-    - **Ingest Latest CISA ICS Medical Advisory**, playbook ingests all the ICS Medical Advisory record.
+    - **Ingest CISA ICS Advisory**, playbook ingests all the ICS Advisory record.
+    - **Ingest CISA ICS Medical Advisory**, playbook ingests all the ICS Medical Advisory record.
     - **Notify for SLA and Report**, playbook notify for BOD 22-01 KEV Alert SLA remaining time and report.
```

Comment:
Need more details here

Reply:
which type of details ?

Reply:
like this pb ingests ICS using abc connector, it creates xyz record and map a,b,c like fields values to help analyst in ....